modules/camera/camera_factory.py

```diff
@@ -19,8 +18,8 @@ class CameraOption(enum.Enum):
 
 
 def create_camera(
-    camera_option: CameraOption, width: int, height: int
-) -> tuple[True, base_camera.BaseCameraDevice] | tuple[False, None]:
+    camera_option: CameraOption, width: int, height: int, config: None
```

Comment:
config type should be `[camera_config.PiCameraConfig | camera_config.OpenCVCameraConfig]`